config/development.json

```diff
@@ -4,6 +4,9 @@
     "client": false,
     "firefoxProxy": false
   },
+  "redux-devtools": {
+    "enabled": false
```

Comment:
Why not just `reduxDevtools: true`? Also camel-cased because we use that for all other options